src/components/ContactsList/ContactsEmptyList.jsx

```diff
@@ -42,17 +47,23 @@ class ContactsEmptyList extends React.Component {
     this.props.hideModal()
     return this.props.showModal(<ContactCardModal id={contact.id} />)
   }
+
   render() {
-    const { hasConnector } = this.state
-    const host = window.location.host
-    const isToutaticeInstance =
-      /\.mytoutatice\.cloud$/.test(host) ||
-      /\.testcloud\.toutatice\.fr$/.test(host) //@TODO Some contexts don't have the google connector available. A better way to do this would be to query the list of available connectors.
     const {
       t,
       showModal,
-      breakpoints: { isDesktop }
+      breakpoints: { isDesktop },
+      client
     } = this.props
+    const { hasConnector } = this.state
+
+    const appStoreUrl = generateUniversalLink({
+      cozyUrl: client.stackClient.uri,
```

Comment:
getStackClient(). (you should have a warning in the console)

Reply:
useClient and not withClient

Reply:
I don't think `client.stackClient` is deprecated. Maybe you thought about `.getClient` https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L1171 or `.client` https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L1166 ? What about it?

Anyway, there's no warning in the console.